content/en/docs/ops/configuration/telemetry/telemetry-v1-vs-v2/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Differences between Telemetry v1 and v2
+description: Tradeoffs and gaps between Telemetry v1 and v2.
+weight: 10
```

Comment:
Who is the audience for this doc?
It reads more like a blog post , i think we should turn it into a blog post.

We can have a paragraph about specific differences / lack of parity on the current telemetry page.

Reply:
Agree, I tried making it not feel like a blog post but it was very difficult not to. I can convert this to a blog if that's more appropriate.

Reply:
Issue is there's no good place to add this information in current docs as the surrounding context is not there. After I added the context, it started to look like a blog post.

Reply:
I don't think we should put specific differences on our current telemetry page. New users won't care -- and it could be more of a distraction than a help.  We'll need somewhere to document the gRPC shim -- and the two should probably be closely-aligned.